experimental/Schemes/FunctionFields.jl

```diff
@@ -388,7 +388,7 @@ canonical_unit(f::VarietyFunctionFieldElem) = f # part of the ring interface tha
 function Base.show(io::IO, KK::VarietyFunctionField)
   io = pretty(io)
   if get(io, :supercompact, false)
-    print(io, "Field")
+    print(io, "Function field")
```

Comment:
In the detailed printing method this is referred to as "Field of rational functions" instead